materializer/nestedfieldselection/README.md

```diff
@@ -0,0 +1,132 @@
+`@materializer` is a great way to seamlessly extend a type with data from another type by executing an operation, sometimes with arguments supplied by the extended type. 
```

Comment:
`@materializer` is a great way to seamlessly extend a type by specifying a field's value is the result of executing an operation

Reply:
not outdated, just seems "from another type" is not technically correct.